R/update-version.R

```diff
@@ -55,3 +54,104 @@ get_date <- function() {
   author_time <- parsedate::parse_iso_8601(Sys.getenv("GIT_COMMITTER_DATE"))
   as.Date(author_time)
 }
+
+fledge_bump_version <- function(desc, which) {
+  version <- desc$get_version()
+  version_components <- get_version_components(version)
```

Comment:
For easier reading of the code:

```suggestion
  version_components <- get_version_components(version)
  dev <- version_components[["dev"]]
  patch <- ...
  ...
```